package.nls.json

```
@@ -29,6 +29,7 @@
     "AWS.codelens.lambda.configEditor": "Edit Debug Configuration (Beta)",
     "AWS.configuration.title": "AWS Configuration",
     "AWS.configuration.profileDescription": "The name of the credential profile to obtain credentials from.",
+    "AWS.configuration.description.ecs.warnRunCommand": "Prompt the user before executing a command in a running ECS container.",
```

we don't usually refer to the user in the third person
```suggestion
    "AWS.configuration.description.ecs.warnRunCommand": "Prompt before executing a command in an ECS container.",
```

---

Phrasing was changed to your suggestion.